packages/jellyfish-api-core/__tests__/category/account/getAccount.test.ts

```diff
@@ -0,0 +1,93 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import waitForExpect from 'wait-for-expect'
+
+describe('masternode', () => {
```

Comment:
for tests that only setup one container you can remove the describe block

Reply:
instead of calling it masternode, probably best to describe the setup e.g.

```suggestion
describe('Account with DBTC', () => {
```